kore/test/Test/Kore/Comparators.hs

```diff
@@ -906,19 +906,23 @@ instance EqualWithExplanation Sort where
     compareWithExplanation = rawCompareWithExplanation
     printWithExplanation = show
 
-instance StructEqualWithExplanation TermLike.Alias where
+instance StructEqualWithExplanation (TermLike.Alias (TermLike Variable)) where
     structFieldsWithNames
-        expected@(TermLike.Alias _ _ _)
-        actual@(TermLike.Alias _ _ _)
+        expected@(TermLike.Alias _ _ _ _ _)
+        actual@(TermLike.Alias _ _ _ _ _)
       =
         [ Function.on (EqWrap "aliasConstructor = ")
             TermLike.aliasConstructor expected actual
         , Function.on (EqWrap "aliasParams = ")
             TermLike.aliasParams expected actual
+        , Function.on (EqWrap "aliasLeft = ")
```

Comment:
As with the `Hashable` instance, `aliasSorts`, `aliasLeft`, and `aliasRight` are intentionally omitted here.